discord/appinfo.py

```diff
@@ -52,9 +55,28 @@ class AppInfo:
         grant flow to join.
     rpc_origins: Optional[List[:class:`str`]]
         A list of RPC origin URLs, if RPC is enabled.
+    summary: Optional[:class:`str`]
+        If this application is a game sold on Discord,
+        this field will be the summary field for the store page of its primary sku
+    verify_key: Optional[:class:`str`]
+        The base64 encoded key for the GameSDK's GetTicket
+    guild_id: Optional[:class:`int`]
```

Comment:
You don't ever set `AppInfo.guild_id`.
```suggestion
    guild: Optional[:class:`Guild`]
```